exec/cnex/module.c

```diff
@@ -1,16 +1,30 @@
-#include "module.h"
+#ifdef _WIN32
+#define _CRT_SECURE_NO_WARNINGS
+#else
+// The following is needed for strdup on Posix platforms.
+#define _POSIX_C_SOURCE 200809L
+#endif
 
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
 
+#ifdef _WIN32
+#include <io.h>
+#define strdup _strdup
+#else
+#include <unistd.h>
+#endif
+
 #include "bytecode.h"
 #include "cell.h"
 #include "exec.h"
+#include "module.h"
```

Comment:
Why did this move? The include for module.h should be first in this file.

Reply:
Opps.  I'm not sure why I changed that.   Fixed.